test/functional/admin/legacy_sitewide_settings_controller_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+
+class Admin::SitewideSettingsControllerTest < ActionController::TestCase
```

Comment:
At the moment it's complaining as there are two test files with the same class name.

If you do this it'll work 

```
class Admin::LegacySitewideSettingsControllerTest < ActionController::TestCase
  tests SitewideSettingsController

```

Reply:
I think we also need to add the preview design system permission to the user in the non-legacy test 

We can just that helper method so it'll be 

```
login_as_preview_design_system_user  :gds_editor
```

Reply:
ahh thanks